t/plugin/openid-connect5.t

```diff
@@ -138,3 +138,77 @@ __DATA__
     }
 --- response_body_like
 hello world
+
+
+
+=== TEST 2: Call to route with locking session storage, no authentication and unauth_action 'deny' should not block subsequent requests on same session
+--- config
+    set $session_storage redis;
+    set $session_redis_uselocking               on;
```

Comment:
why are these two required?

Reply:
To enable the locking session storage, thats what this bug is about

Reply:
oops, well it was too obvious 😂